monitoring/README.md

```diff
@@ -0,0 +1,38 @@
+# ssv-liquidator-monitor Grafana Service
+
+This repository contains the Grafana service for monitoring SSV Liquidator data.
+
+## Prerequisites
+- Docker
+- Docker Compose
+
+## Running the Service
+```shell
+docker-compose up
+```
+
+This command starts two Docker containers:
```

Comment:
This command runs all the services from compose, and one of them is worker itself, please align the readme